op-tests/test-blspy-g2.txt

```diff
@@ -0,0 +1,120 @@
+bls_g2_subtract 0x94b808c39519345a57a625cf8150c640c2e2a404297d97a8b32f13dbe90702c13b130225c3ce309e41ccafc916ff28741169be629c68c4cd713df97cada59e2cee15ed72231602335263eb55bc14e1c7cb023fdb4dc6d6944776332268434c26 0x85de379e95972ab0623b520fc9cf686878d589548c267dc1165354c640c10b7431dfcb9f13691c8726d84789bcf67475196576d627745503b98e594f43b2e2d95c7ed2e1309979862fe2a81825b92075c0ca9a79ae1883ad8fa667ba82e8af41 => 0xa3b4111e01594b6f97d09718e6886940be81697f7e0377546846e14d7298cf201a0f684d7a38da7fbc359ce9ce3b28360043aa7bed503a78119d36638f272c8c6f8979663e3e3ff30889750b37991ae24b0b2d89317bcdfae4b84e71688f6b3c | 11137794
+bls_g2_subtract 0xa3b4111e01594b6f97d09718e6886940be81697f7e0377546846e14d7298cf201a0f684d7a38da7fbc359ce9ce3b28360043aa7bed503a78119d36638f272c8c6f8979663e3e3ff30889750b37991ae24b0b2d89317bcdfae4b84e71688f6b3c 0x84250bba4dc02a41dfbb4913386abb67b2b03823a246aa8d7aab78e547469552746cda9e9da25fc867fe0109219b5b1618d5941e1298231220bf93267ebe984fc70d1969afdf0bdf61886c325d316b77dc4c3516fb02c7827db5f819218c6ab2 => 0x8cf8a6f1b0e2ce982c444f1e1868ce1b6ef7e35eaa233022a593315bd3db29b7224cea5e3028dd457c672063819127be11a7d72a1aa797f2d851a79a8b80f9b496130ea83919c6894e58408795a96bf2b93b6dde20214e0de4166dc065dd4dad | 11137794
+bls_g2_subtract 0x8cf8a6f1b0e2ce982c444f1e1868ce1b6ef7e35eaa233022a593315bd3db29b7224cea5e3028dd457c672063819127be11a7d72a1aa797f2d851a79a8b80f9b496130ea83919c6894e58408795a96bf2b93b6dde20214e0de4166dc065dd4dad 0xaab6a407228638c872bd60e4112a5b4ae1ee15ff5123ae6a762dc97cd2184dc0cf6aee27db0221454c662138c93d369e094cf1ecf824f2c1fa374dd302c33f62d4497cc6cf04c6fcf0f4da811f2f72dec379629a1b370eb746db0b9447ecf74b => 0x80edc25c019da2edda1e30b05b0b9a9a405f5267aeabd1652d66f25397605664913f1895d19386f0730da5479a2c033d044fb5d6c4ccc259c10747528b29e57e9ef0ccf7139c3c27ba5a7718dc68ffd74e6fff5bb8c3193143f6eddc53c20d92 | 11137794
+bls_g2_subtract 0x80edc25c019da2edda1e30b05b0b9a9a405f5267aeabd1652d66f25397605664913f1895d19386f0730da5479a2c033d044fb5d6c4ccc259c10747528b29e57e9ef0ccf7139c3c27ba5a7718dc68ffd74e6fff5bb8c3193143f6eddc53c20d92 0x8e112b9b019c4c83475ad272f96496e85497a689b14bd8fd031a6fb2247217a501d14152d68a17bfc8860baec4cc126013641b9626dacd761aa54cf11c87542433f855a735ca790ae8f154db815b99b0c155065b5169729f522c0698c9f29cd0 => 0x90bea7b7d2d4a969911467ad7199503988e5bd7641b5c6c9129b064ec35c2fda2ffea220f34f6a25cab23e8f5e4ed52b03c6e795949a059d0d9d3106717033a29058662931080a6ce9e9416552968e367e0e846b93433b7d5df827c51253ca11 | 11137794
+bls_g2_multiply 0x96bbcfbca241004ba03968b7433664c1b944245828af7a1ebda771495c90d4816508d65db40b601bce51a3689c251f731699411b56e2172dd82eb96f571a691ba2984f1e4e39a05646ceb9c82d1108e78fe3fdfd94cff4fb447efe498dcecc25 82 => 0xafc1f4f785f8b2122e5e3a001405bb21e13c7292a84e3742448bf614bc3f129a117fbc14c4bbcc5f9e3ed06d6fd867c802a0fe0cd20aab3166985b0659874e61cfb8fb0120c517db30e4bf28dd627b5e2217ba59c4a7447431b626169fa57879 | 10079117
```

Comment:
Add tests for 0 and large scalars.

Reply:
I'll add the case of multiplying with 0 to `test-bls-ops.txt` and large scalars to these generated test cases